tests/unit/test_generate_umm_var.py

```diff
@@ -294,3 +294,78 @@ def test_is_variable_concept_id(self):
 
         with self.subTest('Random string returns False'):
             self.assertFalse(is_variable_concept_id('Random string'))
+
+    @patch('varinfo.generate_umm_var.VarInfoFromNetCDF4')
+    @patch('varinfo.cmr_search.GranuleQuery')
+    @patch('varinfo.generate_umm_var.download_granule')
+    def test_generate_collection_umm_var_config_file(
+        self, mock_download_granule, mock_granule_query, mock_varinfo_from_netcdf4
+    ):
+        """A request with all the necessary information should succeed.
+        This test is moderately end-to-end, but mocks the full
+        download_granule function for simplicity. That function is tested
+        in detail in test_cmr_search.py.
+
+        """
```

Comment:
Probably update this with information for this test.

Reply:
It doesn't look like you updated this comment?

Reply:
I forgot to commit that change. done now